Formula/openimageio.rb

```diff
@@ -0,0 +1,158 @@
+    end
+
+    ENV.append "MY_CMAKE_FLAGS", "-Wno-dev" # stops a warning.
+    ENV.append "MY_CMAKE_FLAGS", "-DUSE_OPENCV=OFF" if build.without? "opencv"
+    ENV.append "MY_CMAKE_FLAGS", "-DUSE_JPEGTURBO=OFF" if build.without? "jpeg-turbo"
+    ENV.append "MY_CMAKE_FLAGS", "-DCMAKE_FIND_FRAMEWORK=LAST"
+    ENV.append "MY_CMAKE_FLAGS", "-DCMAKE_VERBOSE_MAKEFILE=ON"
+    ENV.append "MY_CMAKE_FLAGS", "-DUSE_NUKE=OFF"
+
+    ENV.append "MY_CMAKE_FLAGS", "-DUSE_FFMPEG=OFF" if build.without? "ffmpeg"
+    ENV.append "MY_CMAKE_FLAGS", "-DUSE_LIBRAW=OFF" if build.without? "libraw"
+    ENV.append "MY_CMAKE_FLAGS", "-DUSE_PTEX=OFF" if build.without? "ptex"
+
+    # CMake picks up the system's python dylib, even if we have a brewed one.
+    # Code taken from the Insighttoolkit formula
+    if build.with? "python3"
+      py3ver = Language::Python.major_minor_version "python3"
+      ENV["PYTHONPATH"] = lib/"python#{py3ver}/site-packages"
+      ENV.append "MY_CMAKE_FLAGS", "-DPYTHON_EXECUTABLE='#{`python3-config --prefix`.chomp}/bin/python3'"
+      ENV.append "MY_CMAKE_FLAGS", "-DPYTHON_LIBRARY='#{`python3-config --prefix`.chomp}/lib/libpython#{py3ver}.dylib'"
+      ENV.append "MY_CMAKE_FLAGS", "-DPYTHON_INCLUDE_DIR='#{`python3-config --prefix`.chomp}/include/python#{py3ver}m'"
+    end
+
+    args = ["EMBEDPLUGINS=1"]
+
+    # Download standardized test images if the user throws --with-test.
+    # 90% of the images are in tarballs, so they are cached normally.
+    # The webp and fits images are loose.  Curl them each install.
+    if build.with? "test"
+      d = buildpath
+
+      mkdir d+"webp-images" do
+        curl "https://www.gstatic.com/webp/gallery/[1-5].webp", "-O"
+      end
+      mkdir d+"fits-images"
+      mkdir d+"fits-images/pg93" do
+        curl "https://www.cv.nrao.edu/fits/data/tests/pg93/tst000[1-3].fits", "-O"
+        curl "https://www.cv.nrao.edu/fits/data/tests/pg93/tst000[5-9].fits", "-O"
+        curl "https://www.cv.nrao.edu/fits/data/tests/pg93/tst0013.fits", "-O"
+      end
+      mkdir d+"fits-images/ftt4b" do
+        curl "https://www.cv.nrao.edu/fits/data/tests/ftt4b/file00[1-3].fits", "-O"
+        curl "https://www.cv.nrao.edu/fits/data/tests/ftt4b/file0{09,12}.fits", "-O"
+      end
+
+      resource("j2kp4files").stage { (d+"j2kp4files_v1_5").install Dir["J2KP4files/*"] }
+      resource("tiffpic").stage { (d+"libtiffpic").install Dir["*"] }
+      resource("bmpsuite").stage { (d+"bmpsuite").install Dir["*"] }
+      resource("tgautils").stage { (d+"TGAUTILS").install Dir["*"] }
+      resource("openexrimages").stage { (d+"openexr-images").install Dir["*"] }
+      resource("oiioimages").stage { (d+"oiio-images").install Dir["*"] }
+    end
+
+    args << "USE_OPENGL=OFF"
+    args << "USE_QT=OFF"
+    args << "USE_PYTHON3=ON" if build.with? "python3"
+
+    system "make", *args
+    system "make", "test" if build.with? "test"
+    cd "dist/macosx" do
+      prefix.install "bin", "include", "lib"
```

Comment:
Is there no `make install`?

Reply:
No.

Reply:
I've refactored the install routine to use cmake directly. This also allows using `make install`.